app/controllers/bundles_controller.rb

```diff
@@ -21,6 +24,11 @@ def redirect_if_disabled
 
   def index
     @bundles = Bundle.all
+    respond_with(@bundles)
```

Comment:
The index and show methods for bundles are extremely slow in the API and return a ton of data. (for example, it returns all the details for all the measures) This should probably be trimmed down somehow.